calibration/CTDMOG/CGINS-CTDMOG-13428__20150510.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13428,CC_ptcb1,-0.00025,
 37-13428,CC_ptcb2,0,
 37-13428,CC_ptempa0,179.8349,
-37-13428,CC_ptempa1,-0.06974822,
-37-13428,CC_ptempa2,-5.651446e-07,
-37-13428,CC_wbotc,9.2946e-08,
+37-13428,CC_ptempa1,-6.974823e-02,
+37-13428,CC_ptempa2,-5.651447e-07,
+37-13428,CC_wbotc,9.294638e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT